### PR TITLE
style: image & input components

### PR DESCRIPTION
**Pull request by anik31**
In components folder image.css & input.css contains the css for the image & input components respectively.

docs.html contains the html for all the components.

Deploy preview - https://deploy-preview-3--vacay-ui.netlify.app/docs

### CODE REVIEW

File: components/vacay.css [hunk 1]
```diff
@@ -40,6 +42,7 @@
     --yellow-color-200:#FDE68A;
     --yellow-color-900:#78350F;
     --red-color-200:#FECACA;
+    --red-color-400:#EF4444;
```
Comment (nayyyhaa) [inline components/vacay.css:45]: just a suggestion. you can change naming to capital letters like --RED-COLOR-400 to increase readability. Rest, nice work! 👍

**Review by nayyyhaa — APPROVED** [review]

File: components/vacay.css (hunk 1 above)
Comment (anik31) [reply components/vacay.css:45]: Is it a good practice @nayyyhaa ? I saw in MDN they were using lowercase.

**Review by foolhardy21 — APPROVED** [review]
Looks Nice.

File: components/input.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    align-items: center;
+}
+
+.input label{
+    font-weight: 500;
+    margin: 0.3rem;
+}
+
+input{
+    border: 1px solid var(--lynx-gray-color);
```
Comment (vrx29) [inline components/input.css:19]: better naming can be done for color variables. good format would be --COLOR-PRIMARY like this. I am not able to understand what this lynx is

File: docs.html [hunk 3]
```diff
@@ -106,7 +106,7 @@ <h2>Button</h2>
                 <a href="" class="btn btn-primary-link">Primary</a>
                 <a href=""><i class="btn-icon btn-primary-icon fas fa-shopping-cart"></i></a>
                 <a href="" class="btn btn-primary-icon-text"><i class="btn-icon fas fa-shopping-cart"></i>Add to cart</a>
-                <a class="btn-float-action"><i class="fas fa-plus"></i></a>
+                <a href="" class="btn-float-action"><i class="fas fa-plus"></i></a>
```
Comment (vrx29) [inline docs.html]: dont leave href="". If you doesnt want anything just put # inside href tag.

**Comment by aayushjain-code** [comment]
LGTM clean UI keep working!

File: components/input.css (hunk 2 above)
Comment (anik31) [reply components/input.css:19]: Actually I am using different shades of gray & black, so I will rename them during refactoring.

File: docs.html (hunk 3 above)
Comment (anik31) [reply docs.html]: Will do this

**Comment by anik31** [comment]
Thanks for the reviews @Aayushjain-code @vrx29 @nayyyhaa @foolhardy21

**Pull request merged**